### PR TITLE
contributors, media: add Figma Quick Start Guide

### PR DESCRIPTION
**Pull request by CaitinChen**

### CODE REVIEW

**Comment by gingerkidney** [comment]
LGTM

File: contributors/README.md [hunk 1]
```diff
@@ -111,6 +111,12 @@ See [Code Comment Style](./code-comment-style.md) for details.
 
 See [Commit Message and Pull Request Style](./commit-message-pr-style.md) for details.
 
+### Diagram Style
```
Comment (lilin90) [inline contributors/README.md]: Please also add it to the beginning `## Contents` section in this file.

File: contributors/README.md [hunk 2]
```diff
@@ -111,6 +111,12 @@ See [Code Comment Style](./code-comment-style.md) for details.
...
+
+See [Figma Quick Start Guide](./figma-quick-start-guide.md)
```
Comment (lilin90) [inline contributors/README.md]:
```suggestion
See [Figma Quick Start Guide](./figma-quick-start-guide.md).
```

File: contributors/README.md (hunk 1 above)
Comment (CaitinChen) [reply contributors/README.md]: Added.

Comment (CaitinChen) [reply contributors/README.md]: > Please also add it to the beginning `## Contents` section in this file.

Added.

**Comment by lilin90** [comment]
> it's better to put all the images of figma to a single fold in `media`.

@CaitinChen Agree with this comment because the Figma guide does include many images (18 ones). Please address this comment.

**Comment by CaitinChen** [comment]
@zz-jason @lilin90 Thanks. Created a "figma-guide" folder in "media."

**Pull request merged**